src/components/charts/Vis6.js

```diff
@@ -56,21 +67,33 @@ export const options = {
     },
   },
 }
-
+const URL = 'http://localhost:3000/api/visualization6'
 function Vis6() {
   const [data, setData] = useState([])
   const [datasets, setDatasets] = useState([])
 
   useEffect(() => {
-    setData(DATA)
+    axios
+      .get(URL)
+      .then((response) => {
+        let data = response.data.data
+        setData(data)
+        setDatasets(generateIceCoreData(data))
```

Comment:
This line won't work, since executed right after set data. Put it into other useeffect

Reply:
done